### ISSUE TITLE
Example does not work

### ISSUE THREAD

**Issue by nichtich** [nperez/metacpan-api-tiny#2]
I tried your example to get all releases of author OALDERS, but all modules are returned:

``` perl
my $result = $mcpan->release(
    search => {
        author => "OALDERS AND ",
        filter => "status:latest",
        fields => "author", # name in the example, but anyway
        size   => 1,
    },
);
say $result->{hits}->{hits}->[0]->{fields}->{author}; # not "OALDERS"!
```

**Comment by nperez** [comment]
Yeah, I had some time to look at this this afternoon, and you're right. Unfortunately, I think the API has changed. I also looked at MetaCPAN::API itself and it is now marked as deprecated (DON'T USE). I'll probably not update this to be compatible with MetaCPAN::Client (the new correct client to use).

Patches welcome if you really want this, but I'm not really interested in maintaining this module.
